src/components/RestaurantPreview.tsx

```diff
@@ -0,0 +1,31 @@
+import "@buildo/bento-design-system/index.css";
+import "@buildo/bento-design-system/defaultTheme.css";
+import { Card, Stack, Title, Body, Label } from "@buildo/bento-design-system";
+import { useTranslation } from "react-i18next";
+
+export interface PreviewProps {
+  name: string;
+  image_url: string;
+  rating: number;
+  address: string;
+}
+
+function RestaurantPreview({ name, image_url, rating, address }: PreviewProps) {
+  const { t } = useTranslation();
+  return (
+    <Card elevation="small" borderRadius={8} paddingX={24} paddingY={40}>
+      <Stack space={8}>
+        <img src={image_url} width="70%" height="100%" />
+        <Title size="medium">{name}</Title>
+        <Body size="medium">
+          {address}
+          <Label size="small" color="default">
+            {`${t("Card.Rating")}: ${rating} ★`}
```

Comment:
You can integrate this string's structure in the localized text, look into [interpolation](https://www.i18next.com/translation-function/interpolation).